app/resources/base.html.template

```diff
@@ -17,7 +17,9 @@
 
 <!DOCTYPE html>
 <html dir="{{env.rtl|yesno:"rtl,ltr"}}" lang="{{env.lang}}">
-<head>{% block head %}
+<head>
+  <meta charset="utf-8">
```

Comment:
>Contain a <meta charset="utf-8"> tag as the first child of their <head> tag.

[Guideline](https://www.ampproject.org/docs/tutorials/create/basic_markup
) suggest to put as the first child of head. But ampstart template uses ampbase template so I'll revert this file as well.